src/components/BottomMenuModal/HistoryModal.tsx

```diff
@@ -185,68 +64,59 @@ const HistoryModal = ({ isContentVisible }: HistoryModalProps) => {
     );
   }
 
-  const sortedTransactions = allTransactions.sort(
-    (a, b) => b.timestamp - a.timestamp
+  const transactions = (history as TransactionHistory).results;
```

Comment:
May be worth checking here that `.results` exists before continuing

Reply:
Yes good point, added a '?' to check